Library/Navigation.swift

```diff
@@ -132,6 +132,7 @@ private let allRoutes: [String: (RouteParamsDecoded) -> Navigation?] = [
   "/signup": signup,
   "/projects/:creator_param/:project_param": project,
   "/projects/:creator_param/:project_param/backing/:backing_tab": projectSurvey,
+  "/projects/:creator_param/:project_param/backing/redeem": projectRedeem,
```

Comment:
We don't need this url added separately; it's already covered by line 134.

Reply:
The `:` makes it a parameter name. So /projects/[creator id or name]/[project id or slug]/backing/[any alphanumeric string] will be interpreted as a `projectSurvey` url and be opened in the webview. A malformed url (where the `backing_tab` is something that doesn't exist) would show a `404` in the webview - not ideal but something I'm okay with living with considering how many backing tabs we need to support. Let me know if you disagree, though!

(I updated this fairly recently, in https://github.com/kickstarter/ios-oss/pull/2188/files#diff-001c28f2becf20e1b939e1f60059a4a4dea67c0fc5bd21850ffd7213bfa10390 instead of adding a 3rd url - my hope was that this change would also mean that web will be able to make further changes to the survey without us needing to add local changes to support them. But I don't think it's even in prod yet, so happy to revisit if you'd rather we list the urls out)